features/datachoices/src/main/java/org/opennms/features/datachoices/internal/usagestatistics/UsageStatisticsReporter.java

```diff
@@ -348,6 +376,10 @@ private void setOpenNmsJmxAttributes(UsageStatisticsReportDTO usageStatisticsRep
         if (coreFlowsPersistedObj != null) {
             usageStatisticsReport.setCoreFlowsPersisted((long) coreFlowsPersistedObj);
         }
+        Object flowsPerSecondObj = getJmxAttribute(JMX_OBJ_OPENNMS_FLOWS_PER_SECOND, JMX_ATTR_COUNT);
```

Comment:
I don't see any metric that we expose called `flowsPerSecond`.
We may have to expose this explicitly then we can collect this.

Also, you may want to test anything that you add.
Flows can be generated from here : https://github.com/OpenNMS/udpgen